recipes-bsp/atf/atf-juno_git.bb

```diff
@@ -29,19 +33,27 @@ do_compile[depends] += "u-boot-juno:do_deploy"
 do_compile() {
     oe_runmake \
       CROSS_COMPILE=${TARGET_PREFIX} \
-      all \
```

Comment:
any reason to re-order? it makes diff more difficult to read and see what are the changes.
It also matches the order used in Poplar and allow easily to see what are the differences.

Reply:
It's reordered to match the ones for juno in the op-tee buildfile.

Reply:
if you can keep previous order please. Poplar is going to have optee too so I'll prefer consistency in our recipe over what's in op-tee build file. Next board with ubooot/optee will just have to follow existing recipe practice.